aphrodite/engine/args_tools.py

```diff
@@ -24,96 +24,102 @@
 @dataclass
 class EngineArgs:
     """Arguments for Aphrodite engine."""
-
+    # Model Options
+    device: str = "auto"
+    seed: int = 0
     model: str
-    served_model_name: Optional[Union[List[str]]] = None
     tokenizer: Optional[str] = None
-    skip_tokenizer_init: bool = False
+    revision: Optional[str] = None
+    code_revision: Optional[str] = None
+    tokenizer_revision: Optional[str] = None
     tokenizer_mode: str = "auto"
     trust_remote_code: bool = False
     download_dir: Optional[str] = None
     load_format: str = "auto"
```

Comment:
I would categorize `load_format`, `dtype` and `ignore_patterns` under Load Options.